deliveries/lastic-milestone-3-4.md

```diff
@@ -0,0 +1,218 @@
+
+**Mandatory Guidelines:**
+> It is crucial to follow the [official milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md) closely. Do not remove any sections marked as mandatory or outlined with bold letters or headlines. Optional advisory comments, like this one, can be removed for clarity.
+
+* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/lastic-grant3.md
+* **Milestone Number:** 1 and 2
+
+### Context (Optional)
+Lastic's milestones 3 and 4 aims to enhance the functionality and user experience of the Coretime Chain indexing process through Subsquid, along with providing basic visualization capabilities.
+
+### Deliverables
+This section lists all deliverables for Milestone 3 and 4, as outlined in the original application. Each deliverable includes a link to its implementation, with additional notes provided where applicable.
+
+### Milestone 1: Indexing Coretime Chain with Subsquid - Functioning Indexer and Basic Visualization
+
+This milestone focuses on establishing a robust infrastructure for indexing the Coretime Chain using Subsquid, along with the initial steps towards visualizing Coretime data. Each deliverable is detailed below, complete with relevant links to implementations and additional notes for clarity.
+
+| Number   | Deliverable                                      | Link                                                                                                    | Notes                                                                                                     |
+|----------|--------------------------------------------------|---------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| **0a.**  | License                                          | [subsquid-sdk LICENSE](https://github.com/LasticXYZ/squid-sdk/blob/main/LICENSE) and [subsquid LICENSE](https://github.com/LasticXYZ/subsquid/blob/main/LICENSE) | The project is licensed under GPLv3, ensuring open-source accessibility and compliance. |
+| **0b.**  | Documentation                                    | [subsquid-sdk README.md](https://github.com/LasticXYZ/squid-sdk) and [subsquid README.md](https://github.com/LasticXYZ/squid)| Provides extensive documentation in README and dedicated docs site, facilitating user understanding. |
+| **0d.**  | Error Logging                                    | [Lastic UI/issues](https://github.com/LasticXYZ/LasticUI/issues)                                        | Errors are thoroughly documented as issues on GitHub, set for future resolution. We also set up a community bounty program. |
```

Comment:
Hey @poppyseedDev. The spec for 0d reads "In the event of an error, the application will record detailed error information to an event viewer or log file for diagnostics and troubleshooting", but the deliverable links to your UI repo issues. Can you clarify?

Reply:
If the indexer fails it is usually recorded directly in my subsquid application, which I am unable to open source that account since it has my credit card information link that i am using for running the subsquid indexer this is the link -> [https://app.subsquid.io/squids/rococo-coretime/v3/logs](logs link).
In an event that an error happens I usually write an issue about it directly in the issues tab of our Lastic github repository, or if it's a bigger one I mention it in the Lastic group chat.

Reply:
Okay, I'm guessing that's standard subsquid behavior. The link isn't available to me.